dev/AppNotifications/AppNotificationBuilder/AppNotificationProgressBar.cpp

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "AppNotificationProgressBar.h"
+#include "Microsoft.Windows.AppNotifications.Builder.AppNotificationProgressBar.g.cpp"
+
+namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
+{
+    AppNotificationProgressBar::AppNotificationProgressBar()
+        :m_titleStatus{ BindMode::NotSet },
+         m_statusStatus{ BindMode::NotSet },
+         m_valueStatus{ BindMode::NotSet },
+         m_valueStringOverrideStatus{ BindMode::NotSet }
+    {};
+
+    void AppNotificationProgressBar::Title(winrt::hstring const& value)
+    {
```

Comment:
No check for empty args here or anywhere else?

Reply:
NO, there's a task to handle that in a hollistic manner later. At this point we only check for empty IDs and such but not for normal values.